fix: fix issues with node 15, cross-env, run-p, and run-s

dalelotts opened the pull request:
to install on node 15, run `npm install --force --prefix front-end` to get past the puppeteer peer
dependency warning

fix TFENG-467

kchia commented on back-end/src/reservations/reservations.service.js:
```diff
@@ -133,7 +133,7 @@ function create(newReservation) {
   return knex(tableName)
     .where("reservation_date", date)
-    .andWhereNot("status", "finished")
+    .whereNotIn("status", ["finished", "cancelled"])
```
@dalelotts `andWhereNot` was not working.

dalelotts replied:
Was a test failing? or did you observe this? or both?

dalelotts replied:
Trying to figure out if I screwed up the test or if it was something else?

dalelotts replied:
@kchia ^^

kchia replied:
I observed it.

dalelotts committed:
fix: fix issues with node 15, cross-env, run-p, and run-s

to install on node 15, run `npm install --force --prefix front-end` to get past the puppeteer peer
dependency warning

fix TFENG-467
